### ISSUE TITLE
Change default home page after login

### ISSUE THREAD

**Issue by vanlooverenkoen** [sulu/sulu#6485]
# Problem description
Right now, after login we get to the search page. Because this page is an empty screen with a search bar it is not always clear why we ended up here.
![image](https://user-images.githubusercontent.com/21172855/153215857-ea82af33-7d63-4583-8081-46f2ea125d45.png)

# Proposed solution

It would be nice if we could have an easy to configure `home page` (after login). Taking into account that if we go to a specific page and we are not logged in, we login and get to the previous page instead of the `home page`

**Comment by alexander-schranz** [comment]
@vanlooverenkoen There were already some discussions about adding a "Dashboard" the login screen. We already had some internal discussion about how to achieve that with maybe some widget registry in the admin js and configurable widgets for a user Admin UI. But it currently has no priority yet on our list.

Having another "default page" as a login area does not work as the logged in user maybe could not have access to that area, so the search is currently the only place which all user can see and access. The issue about the dashboard can be found here: https://github.com/sulu/sulu/issues/2815 if you want to add any thoughts about it there.

**Comment by vanlooverenkoen** [comment]
Having another `default page` as the login could indeed cause issues with permissions indeed. but that would be the responsibility of the developer to make sure everybody has access to that page.

**Comment by alexander-schranz** [comment]
@vanlooverenkoen permissions/roles are controlled in sulu over the UI and can be changed by content managers over it, so that is nothing which can be forced by developer in the code which permissions can or can not be set.

**Comment by vanlooverenkoen** [comment]
Yes indeed. What I mean is having the option so a developer could say when using rule: x -> default page is: y

**Comment by sebheitzmann** [comment]
I have tried to change the response of the login in the Symfony Backend, but the login don't honnor the url sended by the login path.
![image](https://user-images.githubusercontent.com/2197902/174974518-855c57c7-a02f-43ea-bc86-a229c88b75b2.png)

It could be possible to just follow the returned url ?

**Comment by alexander-schranz** [comment]
Currently not. The frontend itself does always stay on the current "Frontend Route". Something like you did post `/admin/dashboard/` isn't even a route the frontend know. If you visit sulu you will see it is `/admin/#/` while you see for the frontend only the hash part is important and so it is `/`.
But you must know `/` is not the login route `/`, it is the `Search` route but it just shows the Login Form because you are currently not logged in. So the loggin form is just displayed instead and after login just the login form disappears and render the `Search`. There is no redirect happening here, because you could also be on a page view, then your session did expire and you have unsaved data in your memory. The login form will just be displayed and after the success login it will still show your unsaved data. So that is why sulu isn't doing anything with the `url` returned, because that could end in data loss for the content manager.

Still there are 2 possiblities. The first one is you overwrite the `Search` View. By creating an own Admin class, get the Search view by its name and set your custom view and title to it. We are doing currently the same for our own website.

https://github.com/sulu/sulu/blob/027b90c68353cebe6e267415ad1e0e11d9dee117/src/Sulu/Bundle/SearchBundle/Admin/SearchAdmin.php#L46

Still there is a way you could achieve landing on an own view after the login. You can set another url when the login screen is available. In Sulu Demo an observer already exist which set admin admin login. But instead you could copy that and do a hash replace there: https://github.com/sulu/sulu-demo/blob/master/templates/bundles/SuluAdminBundle/Admin/main.html.twig#L11-L21

```js
if (!window.location.hash || window.location.hash === '#/') {
    window.location.hash = '#/webspaces'
}
```
